robot/Cumulus/resources/NPSP.py

```diff
@@ -413,15 +413,21 @@ def click_managehh_link(self,title):
         locator=npsp_lex_locators['manage_hh_page']['address_link'].format(title)
         self.selenium.get_webelement(locator).click()      
     
-    def select_lightning_checkbox(self,title):
-        """Clicks on a checkbox using field name"""
+    def set_checkbutton_to(self,title,status):
+        """If status is 'checked' then checks the box if its not already checked. Prints a warning msg if already checked
+          'unchecked' then unchecks the box if its not already checked. Prints a warning msg if already unchecked
+        """
         cb_found=False
         locators = npsp_lex_locators["checkbox"].values()
 
         for i in locators:
             locator = i.format(title)
             if self.check_if_element_exists(locator):
-                self.selenium.get_webelement(locator).click()
+                checkbox=self.selenium.get_webelement(locator)
+                if (status == 'checked' and checkbox.is_selected() == False) or (status == 'unchecked' and checkbox.is_selected() == True):
+                    self.selenium.click_element(checkbox)
+                else:
+                    self.builtin.log("This checkbox is already in the expected status", "WARN")     
```

Comment:
"WARN" doesn't seem like the correct level here. It seems to me like "INFO" would be better. Ultimately it's your decision, but warnings generally should be for something unexpected.

Reply:
the scenarios I have used this keyword (so far)should not have this behavior if the org is set up correctly so I wanted to add it as a warning for a manual tester as a heads up in case those are not expected.